QUEST_JOBSTEP.tsv

```diff
@@ -2331,20 +2331,20 @@ QUEST_JOBSTEP_20150323_002330	Reject since it seems too dangerous
 QUEST_JOBSTEP_20150323_002331	Ask him what the gift is
 QUEST_JOBSTEP_20150323_002332	Complete the mission of the Elementalist[Elementalist Advancement]
 QUEST_JOBSTEP_20150323_002333	%s defeated
-QUEST_JOBSTEP_20150323_002334	To prove your determination to be stronger, please collect Glizardon's Leather from the 1st basement of Tenet Cathedral, Galok's Canine from the 1st floor, and Black Powders from Egnome at the 2nd floor.
+QUEST_JOBSTEP_20150323_002334	To prove your determination to be stronger, please collect Glizardon's Leather from the 1st basement of Tenet Chapel, Galok's Canine from the 1st floor, and Black Powders from Egnome at the 2nd floor.
 QUEST_JOBSTEP_20150323_002335	{memo X}Collect the flower alcohol of Doyor from Veya Valley
 QUEST_JOBSTEP_20150323_002336	{memo X}Psychokino offered you to become his assistant if you are interested in this kind of field. Obtain flower alcohol of Doyor from Veya Valley which the Psychokino Master is interested in.
 QUEST_JOBSTEP_20150323_002337	{memo X}You have obtained enough flower alcohol of Doyor from Veya Valley. Go back to the Psychokino Master and hand him the alcohol.
 QUEST_JOBSTEP_20150323_002338	{memo X}Flower Alcohol of Doyor
-QUEST_JOBSTEP_20150323_002339	The Linker Master is emphasizing on the importance of the ability to open one's mind as a linker. To test the competency of linker, please obtain Corylus' Core and Galok's Blood from the first floor of Tenet Cathedral.
+QUEST_JOBSTEP_20150323_002339	The Linker Master is emphasizing on the importance of the ability to open one's mind as a linker. To test the competency of linker, please obtain Corylus' Core and Galok's Blood from the first floor of Tenet Chapel.
 QUEST_JOBSTEP_20150323_002340	Corylus' Core
 QUEST_JOBSTEP_20150323_002341	Blood of Galok
 QUEST_JOBSTEP_20150323_002342	Collect the touch leather of Glizardon
-QUEST_JOBSTEP_20150323_002343	The Hunter Master told Querrel Shooter Master that they could engage in an exchange with Glizardon's tough leather for the book. Obtain Glizardon's tough leather from the 1st basement of Tenet Cathedral.
+QUEST_JOBSTEP_20150323_002343	The Hunter Master told Querrel Shooter Master that they could engage in an exchange with Glizardon's tough leather for the book. Obtain Glizardon's tough leather from the 1st basement of Tenet Chapel.
```

Comment:
The Hunter Master told Querrel Shooter Master -> The Hunter Master told the Quarrel Shooter Master
1st basement of Tenet Chapel -> Tenet Chapel B1